src/github.js

```diff
@@ -21,15 +21,20 @@ module.exports = function(comment: Comment) {
     .join('/');
 
   if (urlPrefix) {
+    let startLine;
+    let endLine;
+
+    if (comment.kind == 'typedef') {
+      startLine = comment.loc.start.line;
+      endLine = comment.loc.end.line;
+    } else {
+      startLine = comment.context.loc.start.line;
+      endLine = comment.context.loc.end.line;
+    }
+
     comment.context.github = {
       url:
-        urlPrefix +
-        fileRelativePath +
-        '#L' +
-        comment.context.loc.start.line +
-        '-' +
-        'L' +
-        comment.context.loc.end.line,
+        urlPrefix + fileRelativePath + '#L' + startLine + '-' + 'L' + endLine,
```

Comment:
I didn't intend to re-format this, fwiw. Looks like a commit hook re-formatted.

Reply:
Yep, that's the intended way - we use `husky` to add a commit hook and then `lint-staged` to run run [prettier](https://github.com/prettier/prettier) on source files. It's a mouthful but it totally eliminates stylistic eslint rules, the computer decides what's best in terms of formatting. And since this changes `comment.context.loc.end.line` to `endLine`, the computer decided that it's short enough to fit on one line, so it did :)